source/info.d

```diff
@@ -28,15 +29,68 @@ immutable Exercise[] exercices =
     {
         name: "intro2",
         path: buildPath("exercises", "intro", "intro2.d"),
-        hint: "No hint yet",
+        hint: "
+            The format string expects something.
+            Add an argument inside the function.".outdent.strip(),
+        type: compile,
+    },
+
+// variables:
+    {
+        name: "variables1",
+        path: buildPath("exercises", "variables", "variables1.d"),
+        hint: "
+            When declaring variables D needs a bit more information than just
```

Comment:
```suggestion
            When declaring variables, D needs a bit more information than just
```

Reply:
Fixed in 40b3db32802f8848528c673504ce44c3efe5b569